### PR TITLE
Added pool cleanup for RBD and RBD mirror related tests

### PR DESCRIPTION
**Pull request by sunilangadi2**
# Description
Some of the RBD and RBD mirror test suites execution keep the pool as it is without proper cleanup of test pools
hence added pool cleanup methods in `finally` clause of test which doesn't exist.

Please include Automation development guidelines. Source of Test case - New Feature/Regression Test/Close loop of customer BZs
<details>

<summary>click to expand checklist</summary>

- [ ] Create a test case in Polarion reviewed and approved.
- [ ] Create a design/automation approach doc. Optional for tests with similar tests already automated.
- [ ] Review the automation design
- [ ] Implement the test script and perform test runs
- [ ] Submit PR for code review and approve
- [ ] Update Polarion Test with Automation script details and update automation fields
- [ ] If automation is part of Close loop, update BZ flag qe-test_coverage “+” and link Polarion test
</details>

### CODE REVIEW

File: tests/rbd/rbd_faster_exports.py [hunk 1]
```diff
@@ -20,82 +21,101 @@ def faster_exports(rbd, pool_type, **kw):
...
-        p.spawn(
-            rbd.exec_cmd,
-            cmd="rbd export {}/{} {}".format(pool, clone, dir_name + "/export9880_2"),
+                config.get("io-total"), pool, image
+            )
         )
-        p.spawn(
-            rbd.exec_cmd,
-            cmd="rbd resize -s {} --allow-shrink {}/{}".format("8G", pool, image),
+        rbd.exec_cmd(cmd="rbd snap create {}/{}@{}".format(pool, image, snap))
```
Comment (pavankumarag) [inline tests/rbd/rbd_faster_exports.py:32]: any reasons why snap create all added in this PR if this is only pool PR ?

**Review by pavankumarag — DISMISSED** [review]
@sunilangadi2 atleast add 1 execution or run log just to be sure the code is working.

File: tests/rbd/rbd_faster_exports.py (hunk 1 above)
Comment (sunilangadi2) [reply tests/rbd/rbd_faster_exports.py:32]: this line is just a continuation of line no 30 from old file, nothing changed anything
added test result for this test changes
http://magna002.ceph.redhat.com/cephci-jenkins/cephci-run-TBASE2/

**Review by Manohar-Murthy — APPROVED** [review]
lgtm.

**Pull request merged**